.github/workflows/cffconvert.yml

```diff
@@ -0,0 +1,33 @@
+name: cffconvert
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'release-*'
+    paths:
+      - CITATION.cff
+  pull_request:
+    branches:
+      - 'master'
+      - 'release-*'
+    paths:
+      - CITATION.cff
+
+permissions:
+  contents: read
+
+jobs:
+  validate:
+    name: "validate"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out a copy of the repository
+        uses: actions/checkout@v2
```

Comment:
```suggestion
        uses: actions/checkout@v4
```

Reply:
I would apply this tweak directly but I think "allow maintainers to push changes" was never activate. Anyway, it can be changed later, too.

Reply:
I've committed the suggestion, thanks for the update.